ddl/failtest/fail_db_test.go

```diff
@@ -333,6 +333,10 @@ func TestRunDDLJobPanicDisableClusteredIndex(t *testing.T) {
 }
 
 func testAddIndexWorkerNum(t *testing.T, s *failedSuite, test func(*testkit.TestKit)) {
+	if variable.DDLEnableDistributeReorg.Load() {
```

Comment:
Can we only keep this test just like dist only one TiDB node cluster instead of skip it

Reply:
The worker count is now controlled through the spmc pool, and the test is mock, presumably to match the pool

Reply:
I see，IMO, the mock test in ut or ft should able to set correct worker numbers as testcase expects, because the ut or ft will no forground workload and spmc pool should not blocke adding worker numbers. @hawkingrei please help to verify this point.

Reply:
https://github.com/pingcap/tidb/issues/40933 
I add an issue, @hawkingrei  please help to handle it？

Reply:
> #40933 I add an issue, @hawkingrei please help to handle it？

Let me do it.